### PR TITLE
TaxToken Formatting

### PR DESCRIPTION
**Pull request by mcmerril**
This PR accomplishes the following:

- Cleans up unused code in comments.
- Moves hard-to-read commends above the line of code.
- Adds periods and capitalization when needed.
- Renames the test name of tax token to TaxToken and $TAX.
- Adds text_TaxToken_something formatting to test cases.
- Updates require statement error codes to reflect new formatting.
- Adds further testing to mint, burn, ad industrtMint/Burn.
- Updated all files aside from the launch deployment test and the treasury files.

### CODE REVIEW

File: src/TaxToken.sol [hunk 1]
```diff
@@ -66,26 +69,26 @@ contract TaxToken {
...
 
-        owner = msg.sender;                                         // The "owner" is the "admin" of this contract.
-        balances[msg.sender] = totalSupplyInput * 10**_decimals;    // Initial liquidity, allocated entirely to "owner".
+        owner = msg.sender;                                         
+        balances[msg.sender] = totalSupplyInput * 10**_decimals;
         maxWalletSize = maxWalletSizeInput * 10**_decimals;
         maxTxAmount = maxTxAmountInput * 10**_decimals;
 
-        // TODO: Add to main-net deployment.
+        // TODO: Add before main-net deployment.
```
Comment (chasebrownn) [inline src/TaxToken.sol:91]: Is it before? Doesnt the contract have to be deployed to call those functions? Dont want this comment to be misleading.

File: src/TaxToken.sol [hunk 2]
```diff
@@ -98,31 +101,31 @@ contract TaxToken {
 
     /// @dev whenNotPausedUni() is used if the contract MUST be paused ("paused").
     modifier whenNotPausedUni(address a) {
-        require(!paused() || whitelist[a], "ERR: Contract is currently paused.");
+        require(!paused() || whitelist[a], "TaxToken.sol::whenNotPausedUni, Contract is currently paused.");
```
Comment (chasebrownn) [inline src/TaxToken.sol:104]: TaxToken.sol::whenNotPausedUni()

File: src/TaxToken.sol [hunk 3]
```diff
@@ -98,31 +101,31 @@ contract TaxToken {
...
-        require(!paused() || whitelist[a], "ERR: Contract is currently paused.");
+        require(!paused() || whitelist[a], "TaxToken.sol::whenNotPausedUni, Contract is currently paused.");
         _;
     }
 
     /// @dev whenNotPausedDual() is used if the contract MUST be paused ("paused").
-    modifier whenNotPausedDual(address from, address to) {
-        require(!paused() || whitelist[from] || whitelist[to], "ERR: Contract is currently paused.");
+    modifier whenNotPausedDual(address _from, address _to) {
+        require(!paused() || whitelist[_from] || whitelist[_to], "TaxToken.sol::whenNotPausedDual, Contract is currently paused.");
```
Comment (chasebrownn) [inline src/TaxToken.sol:110]: TaxToken.sol::whenNotPausedDual()

File: src/TaxToken.sol [hunk 4]
```diff
@@ -98,31 +101,31 @@ contract TaxToken {
...
         _;
     }
 
     /// @dev whenNotPausedTri() is used if the contract MUST be paused ("paused").
-    modifier whenNotPausedTri(address from, address to, address sender) {
-        require(!paused() || whitelist[from] || whitelist[to] || whitelist[sender], "ERR: Contract is currently paused.");
+    modifier whenNotPausedTri(address _from, address _to, address _sender) {
+        require(!paused() || whitelist[_from] || whitelist[_to] || whitelist[_sender], "TaxToken.sol::whenNotPausedTri, Contract is currently paused.");
```
Comment (chasebrownn) [inline src/TaxToken.sol:116]: TaxToken.sol::whenNotPausedTri()

**Review by chasebrownn — APPROVED** [review]
lgtm

File: src/TaxToken.sol (hunk 1 above)
Comment (mcmerril) [reply src/TaxToken.sol:91]: Could go either way, I just thought it made more sense to add that line before we do a main-net deployment

**Pull request merged**